jupyterlab/dashboards/Stocks.ipynb

```diff
@@ -0,0 +1,183 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
```

Comment:
Would you consider using https://github.com/chbrown/nbfilter to wipe out the metadata and the cell outputs?
This are notebooks have better readability in git

Reply:
`pip install nbfilter` and then

`python -m nbfilter.clean < Stocks.ipynb | sponge Stocks.ipynb`

Reply:
Great idea! Done.